apps/dash-financial-report/app.py

```diff
@@ -0,0 +1,66 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+from utils import Header, make_dash_table
+import pandas as pd
+from pages import (
+    overview,
+    pricePerformance,
+    portfolioManagement,
+    feesMins,
+    distributions,
+    newsReviews,
+)
+
+app = dash.Dash(
+    __name__, meta_tags=[{"name": "viewport", "content": "width=device-width"}]
+)
+server = app.server
+
+# Describe the layout/ UI of the app
+app.layout = html.Div(
+    [dcc.Location(id="url", refresh=False), html.Div(id="page-content")]
+)
+
+# Update page
+@app.callback(
+    dash.dependencies.Output("page-content", "children"),
+    [dash.dependencies.Input("url", "pathname")],
+)
+def display_page(pathname):
+    if pathname == "/dash-financial-report/price-performance":
```

Comment:
This is function is totally fine. This is just another alternative solution that I prefer but you don't have to change it 😄 

```Python
    try: 
        lookup_page = {
            "/": overview.create_layout(app),
            "/dash-financial-report": overview.create_layout(app),
            "/dash-financial-report/price-performance": pricePerformance.create_layout(app),
            "/dash-financial-report/portfolio-management": portfolioManagement.create_layout(app),
            "/dash-financial-report/fees": feesMins.create_layout(app),
            "/dash-financial-report/distributions":  distributions.create_layout(app),
            "/dash-financial-report/news-and-reviews": newsReviews.create_layout(app),
            "/dash-financial-report/full-view": (
                overview.create_layout(app),
                pricePerformance.create_layout(app),
                portfolioManagement.create_layout(app),
                feesMins.create_layout(app),
                distributions.create_layout(app),
                newsReviews.create_layout(app),
            ),
        }
        return lookup_page[pathname]
    except KeyError as error:
        # behaves like a 404 page
        return overview.create_layout(app)
```